### PR TITLE
Checkout: update error boundary messages for consistency

### PR DESCRIPTION
**Pull request by michaeldcain**
I missed a typo in the review of #43268. This updates the error boundary messages in Checkout for consistency (and shared translations).

### CODE REVIEW

File: client/my-sites/checkout/checkout-system-decider.js [hunk 1]
```diff
@@ -115,7 +115,7 @@ export default function CheckoutSystemDecider( {
 	if ( 'composite-checkout' === checkoutVariant ) {
 		return (
 			<CheckoutErrorBoundary
-				errorMessage={ translate( 'Sorry. there was an error loading this page' ) }
+				errorMessage={ translate( 'Sorry, there was an error loading this page.' ) }
```
Comment (a8ci18n) [inline client/my-sites/checkout/checkout-system-decider.js:118]: :ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.

File: packages/composite-checkout/src/components/checkout-provider.js [hunk 2]
```diff
@@ -101,7 +101,7 @@ export const CheckoutProvider = ( props ) => {
 	);
 
 	// This error message cannot be translated because translation hasn't loaded yet.
-	const errorMessage = 'Sorry, there was an error loading this page';
+	const errorMessage = 'Sorry, there was an error loading this page.';
```
Comment (sirbrillig) [inline packages/composite-checkout/src/components/checkout-provider.js:104]: The comment about translations here is no longer true. I missed it when I was updating the translations to use `__`, but maybe it's worth fixing in this PR! See other files in this packages for examples of how to get and use `__`.

Comment (michaeldcain) [reply packages/composite-checkout/src/components/checkout-provider.js:104]: Thanks for letting me know. I saw it, but had left it b/c of the comment. I'll fix it up.

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/3859276

Hi @michaeldcain, could you please edit the description of this PR and add a screenshot for our translators? Ideally it'd include this string: `Sorry. there was an error loading this page`

Thank you in advance!

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/3859276).

**Pull request merged**